release_docs/NEWSLETTER.txt

```diff
@@ -3,7 +3,13 @@ HDF5 version 2.0.0 currently under development
 Features included for the next major release:
 ----------------------------------------------------------------------------
 
-*
+      ************ Renamed the option: HDF5_ENABLE_Z_LIB_SUPPORT ************
```

Comment:
Could we break this into two items, with one indicating the rename and one indicating that it now defaults to off? We'd want someone skimming the headlines to catch both of these